Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1974,9 +1976,71 @@ static void activateSessionCleanup(NetworkSessionCocoa& session, const NetworkSe
     m_attributedBundleIdentifierFromPageIdentifiers.add(pageID, parameters.attributedBundleIdentifier());
 }
 
+// FIXME: This and WKURLSessionTaskDelegate are kind of duplicate code.
```

Comment:
Should this be in Bugzilla instead of here? Or is this a "if you make a fix here go make it in the other place" sort of comment?

Reply:
This is a "This code should be removed" kind of comment, but removing the code is more work than should be reasonably done in one PR.